src/payment/payment-methods/authorizenet/__snapshots__/PayPalButton.test.jsx.snap

```diff
@@ -0,0 +1,26 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This file is confusing -> it is in authorize-net folder but contain paypal code ?